nntrainer/compiler/tflite_opnode.cpp

```diff
@@ -65,8 +65,15 @@ void TfOpNode::finalize() {
   auto transform_if = [this](TransformFn &fn, Variables &v) {
     if (fn) {
       auto result = fn(v);
-      NNTR_THROW_IF(result.size() != v.size(), std::invalid_argument)
-        << "result size must match with given variable size";
+      v.resize(result.size());
+      /// basically, result.size() == v.size() except InputLayer because a
+      /// Transpose operator is added for converting nchw to nhwc
+      /// @todo comment out below codes. TfOpNode needs to have LayerNode
+      /// pointer
+      // NNTR_THROW_IF(dynamic_cast<InputLayer>(layer_ptr->getLayer()) ==
+      // nullptr && result.size() != v.size(), std::invalid_argument)
+      //   << "result size must match with given variable size";
+
```

Comment:
This is not a typo error and requires separated discussion or review.

Reply:
It's my mistake.
I will fix right away